airbyte-integrations/connectors/source-file/source_file/spec.json

```diff
@@ -141,24 +133,6 @@
               }
             }
           },
-          {
-            "title": "WebHDFS: HDFS REST API (Untested)",
```

Comment:
should this `spec.json` file also contain SCP as an option? I only see SSH and SFTP